Remove -d flag from 'find'

classicrocker883 opened the pull request:
<!--

Submitting a Pull Request

- Please fill out all sections of this form. You can delete the helpful comments.
- Pull Requests without clear information will take longer and may even be rejected.
- We get a high volume of submissions so please be patient during review.

-->

### Description
This aims to fix some formatting, as well as update comments.

one big imrpovement is **languageImport.py**, so before it would print in the terminal basically garbage, unreable... this organizes the print output to something way better and understandable.  
Also, running `build_all_examples`, `format_code` would give issues.

Mostly, changes are the header comments to the file. like adding the name of the file, and using `"""` for python comments

### Updated/improved other **.sh** scripts

- **build_all_examples**
- - was getting an error with `-ds` in `find` option
- **format_code**
- - using `uncrusitfy [file]` kept repeating, adding `shift` fixed that
- **mfenvs**
- - improved on code
- **mftest**
- - slight update
- **mfutil**
- - improve on code
<!--

Clearly describe the submitted changes with lots of details. Include images where helpful. Initial reviewers may not be familiar with the subject, so be as thorough as possible. You can use MarkDown syntax to improve readability with bullet lists, code blocks, and so on. PREVIEW and fix up formatting before submitting.

-->

### Requirements

<!-- Does this PR require a specific board, LCD, etc.? -->

### Benefits
Improve readability, formatting
<!-- What does this PR fix or improve? -->

### Configurations

<!-- Attach Configurations ZIP and any other files needed to test this PR. -->

### Related Issues

<!-- Does this PR fix a bug or fulfill a Feature Request? Link related Issues here. -->

classicrocker883 committed:
improve on Python scripts

classicrocker883 committed:
improve on Python scripts

classicrocker883 committed:
Update createTemperatureLookupMarlin.py, swap quotes

classicrocker883 committed:
Update createTemperatureLookupMarlin.py, swap quotes

thijstriemstra reviewed (approved):
looks good, exactly how i'd format python code. maybe a followup pr could introduce a (ci) workflow to enforce some of these rules? like triple `"""` for docstrings..

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-August3

classicrocker883 committed:
update/improve format/comments

classicrocker883 committed:
update/improve format/comments

classicrocker883 committed:
improve other scripts

classicrocker883 committed:
improve other scripts

classicrocker883 committed:
update python file headers

classicrocker883 committed:
update python file headers

classicrocker883 committed:
fix typo

classicrocker883 committed:
fix typo

classicrocker883 committed:
exit if cannot find platformio ver

classicrocker883 committed:
exit if cannot find platformio ver

classicrocker883 committed:
simplify validate_pio

classicrocker883 committed:
simplify validate_pio

classicrocker883 committed:
quick changes

classicrocker883 committed:
quick changes

classicrocker883 committed:
use naming convention

classicrocker883 committed:
use naming convention

classicrocker883 committed:
more changes - msg instead of str

classicrocker883 committed:
more changes - msg instead of str

classicrocker883 committed:
Merge branch 'bugfix-2.1.x-August3' of https://github.com/classicrocker883/Marlin into bugfix-2.1.x-August3

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-August3

classicrocker883 committed:
Update schema.py

classicrocker883 committed:
Update schema.py comments

classicrocker883 committed:
solve Inefficient regular expression
This part of the regular expression may cause exponential backtracking on strings starting with '0,' and containing many repetitions of ' 0,'.
CodeQL

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-August3

classicrocker883 committed:
Merge branch 'MarlinFirmware:bugfix-2.1.x' into bugfix-2.1.x-August3

classicrocker883 committed:
Update mfenvs formatting

classicrocker883 committed:
Update mftest formatting

classicrocker883 committed:
update formatting (comments)

classicrocker883 committed:
more idiomatic

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-August3

classicrocker883 committed:
updates - remove spaces

classicrocker883 committed:
Use isinstance() rather than type() for a typecheck.
Codacy

classicrocker883 committed:
codacy fixes
pioutil.py - If is too complex (MC0001)
lerdge.py - Consider using enumerate instead of iterating with range and len
validate_boards.py - last_groun is unused, typo? (last_group)

classicrocker883 committed:
improve mftest/mfenvs, fix pioutil.py,
add "clean" option for mftest

classicrocker883 committed:
update formatting, remove space

thinkyhead committed:
-s lexical sort only, -d may be default anyway

thinkyhead committed:
Merge branch 'bugfix-2.1.x' into pr/28013

thinkyhead committed:
too much

classicrocker883 commented:
unfortunately **build_all_examples** does not work
```r
=====================
Proceed with builds...
=====================
find: unknown predicate `-s'
Exiting
```

thinkyhead commented:
We want sorting, and `| sort` simply doesn't produce the same lexical sorting. The `-d` option can be removed because according to the manpage it is the default behavior for the BSD version of `find` on the systems of the core developers -- who actually use these tools.

thinkyhead commented:
This is the order we're looking for:

<details><summary>`$ build_all_examples --nobuild --purge`</summary>
```
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c 3DFabXYZ/Migbot
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c 3DMatik/XL
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c ADIMLab/Gantry v1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c ADIMLab/Gantry v2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Alfawise/U20
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Alfawise/U20-bltouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AliExpress/CL-260
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AliExpress/UM2pExt
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/A2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/A2plus
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/A6
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/A8
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/A8plus
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/A9
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/E10
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/E16/BTT SKR 1.3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/E16/Stock
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/ET4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/ET4-Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/ET4+
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/ET4X
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/ET5
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/ET5-Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Anet/ET5X
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AnyCubic/Chiron
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AnyCubic/i3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AnyCubic/i3 Mega/Trigorilla AVR
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AnyCubic/i3 Mega/Trigorilla Pro STM32
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AnyCubic/Mega Zero/Anycubic V1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AnyCubic/Mega Zero/BigTreeTech SKR Mini MZ V1.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AnyCubic/Mega Zero 2.0/Anycubic V1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c AnyCubic/Vyper
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c ArmEd
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Artillery/Genius/BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Artillery/Genius/V1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Artillery/Genius Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Artillery/Hornet
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Artillery/Sidewinder X1/0.9 BMG - E3D V6
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Artillery/Sidewinder X1/V1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Artillery/Sidewinder X2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Azteeg/X5GT
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BIBO/TouchX/cyclops
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BIBO/TouchX/default
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BIBO/TouchX/default - BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BIQU/B1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BIQU/B1 SE
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BIQU/B1 SE Plus
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BIQU/BX
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BIQU/Hurakan
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BIQU/Thunder Standard
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BQ/Hephestos
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BQ/Hephestos_2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c BQ/WITBOX
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Cartesio
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c CNC/miniRambo
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Copymaster3D/300
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Copymaster3D/400
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Copymaster3D/500
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10/BigTreeTech SKR Mini E3 2.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10/BigTreeTech SKR Mini E3 3.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10/CrealityV1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 Max
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 Mini/BigTreeTech SKR Mini E3 2.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 Mini/BigTreeTech SKR Mini E3 3.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 Mini/CrealityV1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 Mini/MEEB-3DP
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 S4/CrealityV1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 S5/BigTreeTech SKR Mini E3 1.2 with TFT35 E3 V3.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 S5/BigTreeTech SKR Mini E3 v3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 S5/CrealityV1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 S5/CrealityV2.2 - BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 S5/CrealityV427 - BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 V2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 V3/BigTreeTech SKR Mini E3 v3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10 V3/Stock
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10S/BigTreeTech SKR 1.4 Turbo TMC2209
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10S/BigTreeTech SKR 2.0 TMC2209
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10S/BigTreeTech SKR Mini E3 2.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10S/BigTreeTech SKR Mini E3 3.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10S/BigTreeTech SKR Pro v1.2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10S/CrealityV1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10S/CrealityV1 - BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-10S Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-20/RepRapWorld Minitronics20
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-20/Stock
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-20 Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-30 PrintMill
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-6 SE
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/CR-8
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-2 Pro/CrealityV24S4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-2 Pro/CrealityV423
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech E3 RRF
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech Manta E3 EZ
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech SKR 1.4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech SKR E3 Turbo/2-into-1 Hotend
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech SKR E3 Turbo/Dual Z
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech SKR E3 Turbo/Single Extruder
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech SKR E3-DIP V1.1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech SKR Mini E3 1.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech SKR Mini E3 1.2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech SKR Mini E3 2.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/BigTreeTech SKR Mini E3 3.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/Creality free-runs
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/CrealityV1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/CrealityV422
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/CrealityV427
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/FYSETC Cheetah/base
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/FYSETC Cheetah/BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/FYSETC Cheetah 1.2/base
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/FYSETC Cheetah 1.2/BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/FYSETC Cheetah 2.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/MEEB-3DP
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/MKS Robin E3/V1.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/MKS Robin E3/V1.1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/MKS Robin E3P
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3/TH3D EZBoard Lite V2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Max
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Max Neo
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Neo
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/BigTreeTech SKR 1.4 Turbo
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/BigTreeTech SKR Mini E3 1.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/BigTreeTech SKR Mini E3 1.2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/BigTreeTech SKR Mini E3 2.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/BigTreeTech SKR Mini E3 3.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/Creality free-runs
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/CrealityV1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/CrealityV422
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/CrealityV427
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 Pro/CrealityV427 - BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 S1/STM32F1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 S1/STM32F4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 S1 Plus
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 S1 Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/BigTreeTech SKR E3 Turbo/CrealityUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/BigTreeTech SKR E3 Turbo/MarlinUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/BigTreeTech SKR Mini E3 2.0/CrealityUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/BigTreeTech SKR Mini E3 2.0/MarlinUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/BigTreeTech SKR Mini E3 3.0/CrealityUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/BigTreeTech SKR Mini E3 3.0/MarlinUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/CrealityV422/CrealityUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/CrealityV422/MarlinUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/CrealityV427/CrealityUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2/CrealityV427/MarlinUI
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2 Neo/BigTreeTech SKR Mini E3 3.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-3 V2 Neo/CrealityV422
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5/BigTreeTech SKR E3 Turbo
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5/BigTreeTech SKR E3 Turbo with BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5/BigTreeTech SKR Mini E3 1.2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5/BigTreeTech SKR Mini E3 2.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5/BigTreeTech SKR Mini E3 3.0 with BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5/Creality free-runs
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5/CrealityV1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5/CrealityV427
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5/CrealityV427 with BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Plus/BigTreeTech SKR 3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Plus/CrealityV1 (ORIGIN UI)
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Plus/CrealityV1 (RELOADED UI)
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Pro/BigTreeTech SKR E3 Turbo
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Pro/BigTreeTech SKR Mini E3 1.2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Pro/BigTreeTech SKR Mini E3 2.0 with BLTouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Pro/BigTreeTech SKR Mini E3 3.0 with BLTouch-HighTemp
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Pro/CrealityV1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Pro/CrealityV422
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 Pro/CrealityV427
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-5 S1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Creality/Ender-6
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c CTC/A13
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c CTC/Bizer
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c CTC/i3 2560 Rev A/no probe
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Dagoma/Disco Ultimate
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Daycom/3DP-100
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Anycubic/Kossel
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Anycubic/Kossel Linear Plus
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Anycubic/Predator
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Dreammaker/Overlord
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Dreammaker/Overlord_Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/FLSUN/auto_calibrate
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/FLSUN/kossel
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/FLSUN/kossel_mini
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/FLSUN/Q5-nano_v1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/FLSUN/Q5-nano_v2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/FLSUN/QQ-S
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/FLSUN/QQS-Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Geeetech/G2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Geeetech/G2Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Geeetech/G2S
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Geeetech/G2SPro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Geeetech/Rostock 301
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/generic
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Hatchbox_Alpha
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/kossel_clear
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/kossel_mini
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/kossel_pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/kossel_xl
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Malyan M300
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/MKS/SBASE
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Tevo Little Monster
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c delta/Velleman/K8800
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c E3D/BigBox Dual
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c EasyThreeD/ET4000PLUS
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Eazao/Zero
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Einstart-S
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Elegoo/Neptune 2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Eryone/Thinker SE
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Eryone/Thinker V2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c EXP3D/Imprimante multifonction
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Felix/DUAL
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Felix/Single
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FlashForge/Creator 2X
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FlashForge/CreatorPro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FlyingBear/FlyingBearGhost_v3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FlyingBear/P902
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FlyingBear/P905H
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FoamCutter/generic
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FoamCutter/rcKeith
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FolgerTech/FT-5 R2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FolgerTech/i3-2020
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Formbot/Raptor
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FYSETC/AIO_II
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FYSETC/F6_13
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c FYSETC/S6
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c gCreate/gMax1.5+
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A10
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A10D
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A10M/Stock LCD
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A10M/YHCB2004_V4.1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A10PRO
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A10T/Stock LCD
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A10T/YHCB2004_V4.1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A20
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A20M
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A20T
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A30
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A30M
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/A30T
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/D200
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/Duplicator5
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/E180
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/GT2560
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/I3_Pro_X-GT2560
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/M201
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/Me_creator
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/Me_ducer
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/MeCreator2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/PI3A PRO
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/Prusa i3 Pro B/bltouch
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/Prusa i3 Pro B/noprobe
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/Prusa i3 Pro C
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Geeetech/Prusa i3 Pro W
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Hictop/3DP-17
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c HMS434
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Infitary/i3-M508
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Intamsys/FunmatHT 4988
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c JGAurora/A1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c JGAurora/A3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c JGAurora/A5
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c JGAurora/A5S
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c JGAurora/Magic
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Kingroon/KP3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Kingroon/KP3S
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Kingroon/KP3S_Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Kingroon/KP5L-DIY
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Labists/ET4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c linear_axes/Octopus 8 LINEAR_AXES
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c linear_axes/RAMPS 5 LINEAR_AXES
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Longer/LK5
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c LulzBot/TAZ4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c MakerBot/Replicator Dual
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c MakerBot/Replicator Single
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c MakerFarm/Pegasus 12
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c MakerParts
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c MakerTech3D/Proforge2sDual
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c makibox
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Malyan/M150
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Malyan/M180
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Malyan/M200
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Materia101
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c MBot/Cube
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Micromake/C1/basic
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Micromake/C1/enhanced
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Mks/Robin
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Mks/Robin_Lite3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Mks/Robin_Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Mks/Sbase
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Modix/Big60
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Nextion
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Opulo/Lumen_REV3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Opulo/Lumen_REV4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Opulo/Lumen_REV5
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c ORD Bot/Hadron/Sanguinololu
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Ortur 4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Polargraph
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Printrbot/PrintrboardG2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Printrbot/Simple Metal RevD
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Printrbot/Simple_Metal_RevF6
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Prusa/MK3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Prusa/MK3S-BigTreeTech-BTT002
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Qidi/Qidi 1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Raiscube/A8r
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c RapideLite/RL200
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Renkforce/RF100
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Renkforce/RF100v2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Renkforce/RF100XL
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c RepRapPro/Huxley
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c RepRapWorld/Megatronics
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c RepRapWorld/Minitronics11
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c RepRapWorld/Minitronics20/CNC
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c RigidBot
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Robo3D/R1+
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Rolohaun/Rook MK1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c SCARA/Morgan
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c SCARA/MP_SCARA
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Simax/Mi-M200
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Simulator
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-01/BigTreeTech SKR E3 Turbo
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-01/BigTreeTech SKR Mini E3 2.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-01/BigTreeTech SKR Mini E3 3.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-01/CrealityV22
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-01/CrealityV221
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-02
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-03
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-05
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-06
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sovol/SV-06 Plus
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c STM32/Black_STM32F407VET6
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c STM32/STM32F103RE
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sunlu/S8
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Sunlu/T3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tevo/Michelangelo
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tevo/Nereus
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tevo/Tarantula Pro/MKS Gen_L
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tevo/Tarantula Pro/MKS SGen_L
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tevo/Tornado/V1 (MKS Base)
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tevo/Tornado/V2 (MKS Gen_L)
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tinkerine/Ditto Pro/RAMPS 1.4
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c TinyBoy2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c TPARA/AXEL_TPARA
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/D01/CXY-v6-191017 board
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/X1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/X3A
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/X5S
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/X5S-2E/Green Board (Chitu V5)
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/X5S-2E/Red Board (Tronxy 3.1.0)
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/X5SA
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/X5SA-2E
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/XY100
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Tronxy/XY2-Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Turnigy/Fabrikator
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c tvrrug/Round2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Two Trees/BlueR/BlueR V1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Two Trees/BlueR/BlueR V2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Two Trees/BlueR/BlueR V3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Two Trees/BlueR Plus
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Two Trees/Sapphire Plus/Sapphire Plus V2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Two Trees/Sapphire Plus/Sapphire Plus V2.1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Two Trees/Sapphire Pro
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Two Trees/SP-5_V3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c UltiMachine/Archim1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c UltiMachine/Archim2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Ultimaker/Ultimaker 2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Ultimaker/Ultimaker 2+
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Ultimaker/Ultimaker Original (1.5.7)
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Ultimaker/Ultimaker Original Plus (2.1.1)
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Ultimaker/Ultimaker Original Plus DXU
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Velleman/K8200
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Velleman/K8200 Upgraded
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Velleman/K8400/Dual-head
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Velleman/K8400/Single-head
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c VORONDesign
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Voxelab/Aquila/BigTreeTech SKR Mini E3 2.0
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Wanhao/Duplicator 4S
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Wanhao/Duplicator 6
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Wanhao/Duplicator i3 2.1
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Wanhao/Duplicator i3 Mini
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Wanhao/Duplicator i3 Plus
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c WASP/PowerWASP
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Weedo/Tina2/V2
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Weedo/Tina2/V3
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Weistek/wt150
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Zonestar/P802M
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Zonestar/P802QR2/BigTreeTech SKR 1.4 Turbo (A4988)
[DRYRUN] build_example -b ./.pio/build-import-2.1.x -c Zonestar/P802QR2/BigTreeTech SKR 1.4 Turbo (TMC2209)
```
</details>

thinkyhead commented:
The precise order doesn't matter, especially. It just has to be consistent from build to build so one can resume in the middle after a cancel or fail. Changes pushed in 122dfa13a2.
